en/04_Changelogs/5.0.0.md

```diff
@@ -60,6 +61,35 @@ This is a major release and as a result there are a number of breaking API chang
     - `SilverStripe\ORM\Connect\Query`
   - Removed `SilverStripe\ORM\DataList::getGenerator()` (use `getIterator()` instead)
   - Removed the `SilverStripe\ORM\Map_Iterator` class. `SilverStripe\ORM\Map` now uses a generator instead.
+- Removed various API related to SwiftMailer
+  - Removed the following classes and interfaces
+    - `SilverStripe\Control\Email\Mailer`
+    - `SwiftMailer`
+    - `SwiftPlugin`
+  - Removed the following methods from `SilverStripe\Control\Email\Email`, in most instances these have been replaced with equivalent functionality:
+    - `addAttachment()`
+    - `addAttachmentFromData()`
+    - `hasPlainPart()`
+    - `invalidateBody()`
+    - `findPlainPart()`
+    - `generatePlainPartFromBody()`
+    - `getBCC()`
+    - `getCC()`
+    - `getFailedRecipients()`
+    - `getFrom()`
+    - `getReturnPath()`
+    - `getSender()`
+    - `getSwiftMessage()`
+    - `getTo()`
+    - `setSwiftMessage()`
+    - `setFailedRecipients()`
+    - `BaseURL()`
+    - `IsEmail()`
```

Comment:
For any of these which do have a new replacement, mention it here.
e.g. "`getTo()` replaced with `to()`"

This makes it a lot easier for people to know what changes they need to make.